src/firefly/js/api/ApiHighlevelBuild.js

```diff
@@ -214,15 +222,86 @@ function buildImagePart(llApi) {
 
     const {RequestType}= llApi.util.image;
 
+
+    /**
+     * @summary object for web plot request
+     * @description Below is a list of predefined parameters available for web plot request.
+     * Some parameters control how to get an image, a image can be retrieved from a service, a url, of a file on the server.
+     * Others control the zoom, stretch, and color, title, and default overlays. There are also parameters to pre-process an
+     * image, such as crop, rotate or flip.
+     * @typedef {object} WebPlotParams
```

Comment:
I would really like that to be calls a `WebPlotRequest` as well.  But this is a general challenge with have in JavaScript when you want to make a class out of a simple object.  I guess `WebPlotParams` is as good a name as any.